dataplane/common/Cargo.toml

```diff
@@ -14,4 +14,4 @@ default = []
 user = [ "aya" ]
 
 [dependencies]
-aya = { workspace = true, optional=true }
+aya = { version = "0.13.1", optional=true }
```

Comment:
It may be reasonable to remove the aya dependencies at the workspace level as you did, as we wouldn't expect eBPF code anywhere but the dataplane crate. That said, just note that you did leave the base `aya` dependency in the workspace Cargo.toml.

Reply:
good catch!

Reply:
i guess aya is not a linux only crate, but related crates like aya-log are.